pyproject.toml

```diff
@@ -0,0 +1,36 @@
+[build-system]
+requires = ['setuptools']
+build-backend = 'setuptools.build_meta'
+
+[tool.setuptools]
+packages = ['botbot']
+
+[project]
+name = 'botbot'
+version = '0.5.4'
+description = 'A meta-bot for Euphoria.'
+authors = [
+  { name='Rishov Sarkar' },
```

Comment:
Feel free to add yourself too!

Reply:
Blast from the past :sweat: hey y'all :wave: